scilpy/io/utils.py

```diff
@@ -217,11 +217,21 @@ def add_overwrite_arg(parser):
         help='Force overwriting of the output files.')
 
 
-def add_force_b0_arg(parser):
-    parser.add_argument('--force_b0_threshold', action='store_true',
-                        help='If set, the script will continue even if the '
-                             'minimum bvalue is suspiciously high ( > {})'
-                        .format(DEFAULT_B0_THRESHOLD))
+def add_skip_b0_validation_arg(parser, b0_tol_name='--b0_threshold'):
```

Comment:
I think I prefer `add_skip_b0_check_arg`, since the function is called `check_b0_threshold`.

Reply:
So `--skip_b0_check`. Also shorter to write ;)